corehq/toggles.py

```diff
@@ -782,7 +782,7 @@ def _commtrackify(domain_name, toggle_is_enabled):
 NON_COMMTRACK_LEDGERS = StaticToggle(
     'non_commtrack_ledgers',
     "Enable ledgers for projects not using Supply.",
-    TAG_EXPERIMENTAL,
+    TAG_SOLUTIONS,
```

Comment:
@sheelio I'm not sure `SOLUTIONS` is the right tag for this feature. It was my best guess, otherwise `CUSTOM` maybe?

Reply:
I think Custom for now as well (enikshay)